client/modules/IDE/components/Header/Toolbar.jsx

```diff
@@ -35,34 +30,6 @@ const Toolbar = (props) => {
   const dispatch = useDispatch();
 
   const { t } = useTranslation();
-  const { changeSketchName, canEditProjectName } = useSketchActions();
-
-  const projectNameInputRef = useRef();
-  const [nameInputValue, setNameInputValue] = useState(project.name);
-
-  function handleKeyPress(event) {
-    if (event.key === 'Enter') {
-      dispatch(hideEditProjectName());
-      projectNameInputRef.current?.blur();
-    }
-  }
-
-  function handleProjectNameClick() {
-    if (canEditProjectName) {
-      dispatch(showEditProjectName());
-      setTimeout(() => {
-        projectNameInputRef.current?.focus();
-      }, 140);
-    }
-  }
-
-  function handleProjectNameSave() {
-    const newName = nameInputValue;
-    if (newName.length > 0) {
```

Comment:
FYI it will now use the 0-length behavior from the `EditableInput`, which is to close the input and keep the name from before.

Reply:
is it okay to leave it like this?

Reply:
yes.